runner/test_runners.py

```
@@ -0,0 +1,248 @@
+                {"element_index": 1, "role": "AXButton", "label": "Send"},
+                {"element_index": 2, "role": "AXMenuBarItem", "label": "File"},
+                {"element_index": 3, "role": "AXSecureTextField", "label": "Password"},
+            ]
+        }
+        with mock.patch.object(cua_exec, "cua", return_value=fake):
+            elements, err = cua_exec.snapshot(1, 1)
+        self.assertEqual(err, "")
+        labels = [e["label"] for e in elements]
+        self.assertIn("Send", labels)
+        self.assertNotIn("File", labels)  # macOS menu bar never reaches the model
+        self.assertTrue(any("redacted" in l for l in labels))
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)